test/charms/test_main/src/charm.py

```diff
@@ -20,7 +20,7 @@
 
 sys.path.append('lib')
 
-import ops  # noqa: E402 (module-level import after non-import code)
+import ops
```

Comment:
Fwiw, I do not know why this is no longer necessary, and haven't bothered to dig into it. My assumption is that ruff ignores code that manipulates `sys.path` when doing this check.